plexapi/collection.py

```diff
@@ -116,6 +117,36 @@ def listType(self):
         else:
             raise Unsupported('Unexpected collection type')
 
+    def visibilities(self):
```

Comment:
This method would return a `HubManage` object for the collection.

```python
key = '/hubs/sections/%s/manage?metadataItemId=%s' % (self.section().key, self.ratingKey)
data = self._server.query(key)
try:
    return self.findItems(data, cls=HubManage)[0]
except IndexError:
    psss  # or raise NotFound?
```
No need to cache it.